packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -463,6 +464,84 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     });
   }
 
+  @override
+  Future<void> handlePointerEventPack(List<PointerEventPack> records) {
+    // hitTestHistory is an equivalence of _hitTests in [GestureBinding]
+    final Map<int, HitTestResult> hitTestHistory = <int, HitTestResult>{};
+    return TestAsyncUtils.guard<void>(() async {
+      final DateTime startTime = binding.clock.now();
+      for(final PointerEventPack pack in records) {
+        final Duration timeDiff = pack.timeStamp - binding.clock.now().difference(startTime);
+        if (timeDiff.isNegative) {
+          // Flush all past events
+          // maybe trigger a warning
+          for (final PointerEvent event in pack.events) {
+            _handlePointerEvent(event, hitTestHistory);
+          }
+        }
+        else {
+          await binding.executeLater(timeDiff, () async {
+            for (final PointerEvent event in pack.events) {
+              _handlePointerEvent(event, hitTestHistory);
+            }
+          });
+          // TODO(CareF): delete the if sentence when
+          // https://github.com/flutter/flutter/issues/60739 is fixed
+          if (binding is LiveTestWidgetsFlutterBinding &&
+              (binding as LiveTestWidgetsFlutterBinding).framePolicy ==
+              LiveTestWidgetsFlutterBindingFramePolicy.fullyLive) {
+            continue;
+          }
+          await binding.pump();
```

Comment:
I'm not 100% sure when I need pump. Another possibility is to pump in the beginning of this else clause and in the end of this guard async call.

Reply:
For now, I think we can probably `assert(...framePolicy  == ...fullyLive/benchmarkLive)`and remove `pump` as this is probably only used in our tests. If in the future we want test something with some other `framePolicy `, we can then figure out where to put `pump` with the guidance of those use cases.

Reply:
`framePolicy` requires `LiveTestWidgetsFlutterBinding`, which is not what we use for unit test. And my unit test also depends on this. If I have to change the behavior here I would rather just `pump`, which is what we are already doing for `WidgetTester.drag` anyway